lib/licensee/project.rb

```diff
@@ -2,17 +2,6 @@ class Licensee
   class Project
     attr_reader :repository
 
-    # Array of file names to look for potential license files, in order
-    # Filenames should be lower case as candidates are downcased before comparison
-    LICENSE_FILENAMES = %w[
```

Comment:
You're 100% right. Opened https://github.com/benbalter/licensee/pull/36 with a failing test.